node/src/main/scala/com/wavesplatform/utils/UtilApp.scala

```diff
@@ -251,14 +258,14 @@ object UtilApp {
   private[this] object IO {
     def readInput(c: Command): Array[Byte] = {
       val inputStream = c.inputData match {
-        case Some(value) =>
-          new ByteArrayInputStream(value.getBytes)
+        case Input.StdIn | Input.File("-") =>
```

Comment:
Dedicated types for input sources is a good idea, but why are bot `File("-")`and `StdIn` used for stdin?